### DISCUSSION TITLE
MacOS PAGViewer无法使用

### DISCUSSION THREAD

**Discussion by wangwanjie** [Tencent/libpag#2222] [General] [answered]
mac上 PAGViewer 本来可以使用的，提示我可以升级，点了升级就打不开了，架构都不对了，我本来是arm64的
➜ lipo -info /Applications/PAGViewer.app/Contents/MacOS/PAGViewer
Non-fat file: /Applications/PAGViewer.app/Contents/MacOS/PAGViewer is architecture: x86_64

**Comment by zhalice2011** [comment, marked as answer]
安装一下历史版本看看

https://pag.art/docs/pag-download.html

**Reply by wangwanjie** [reply]
谢谢，历史版本正常使用了
